<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseDialog</class>
 <widget class="QWidget" name="chooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/WAQ.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>五子棋</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>211</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainvLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="chooseLab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请选择游戏模式:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="secondLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="AImodeBtn">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>人机对战模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HumanmodeBtn">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>人人对战模式</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
